src/compute/cast/mod.rs

```diff
@@ -36,6 +36,29 @@ pub use primitive_to::*;
 pub use timestamps::*;
 pub use utf8_to::*;
 
+/// options defining how Cast kernels behave
+#[derive(Clone, Copy, Debug)]
+pub struct CastOptions {
```

Comment:
given that we now have the `cast` and `wrapped_cast`, we can keep this private and only keep the others as the public surface?